### ISSUE TITLE
Accessibility: Hoverable

### ISSUE THREAD

**Issue by ferllings** [atomiks/tippyjs#1104]
Hello,

On non-interactive mode, the tooltip fails WCAG 1.4.13 criteria.
https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html

Using `interactive=true` would allow the tooltip to be hoverable, but it removes `aria-describedby` attribute and incorrectly adds `aria-expanded`.

I'm trying to see, if I can change the default behaviour using the tippy Lifecycle Hooks, but it would be good if this is implemented by default.

**Comment by ferllings** [comment]
Closing as already answered:
https://github.com/atomiks/tippyjs/issues/750
